### PR TITLE
fix issues 10403

### LINKED ISSUE #10403: Add parse & rewrite test case for postgresql start/end transaction
After PR #10402 is merged, we need to add some test cases for postgresql `start/end transaction` statements, including parse result tests and rewrite tests. 

If you are a newbie, you can use the `SQLParserParameterizedTest` and `ShardingSQLRewriterParameterizedTest` classes as the entrance to understand the logic of parsing tests and rewriting tests.

If you are interested, please leave a comment below. BTW, If you need any help, please ping me.

### PR DESCRIPTION
**Pull request by sizzlecar**
Fixes #10403.

### CODE REVIEW

**Comment by tristaZero** [comment]
@sizzlecar Thanks for this PR, let @strongduanmu give a look.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/tcl/end.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-parser-test-cases>
+    <commit sql-case-id="end" />
+    <commit sql-case-id="endPrepare" />
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/tcl/end.xml:21]: @sizzlecar Please keep a uniform style for naming sql-case-id. I think `end_prepare` is better, and please check other sql-case-id naming problems.

Comment (sizzlecar) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/tcl/end.xml:21]: ok

**Comment by sizzlecar** [comment]
@strongduanmu I have modified as required

**Review by strongduanmu — APPROVED** [review]

**Comment by strongduanmu** [comment]
> @strongduanmu I have modified as required

@sizzlecar Thank you for your efforts and welcome to be a contributor.

**Comment by sizzlecar** [comment]
> > @strongduanmu I have modified as required
> 
> @sizzlecar Thank you for your efforts and welcome to be a contributor.

Thanks for the guidance and hope to bring more contributions

**Pull request merged**